docs/api/covidcast-signals/safegraph-patterns.md

```diff
@@ -0,0 +1,45 @@
+---
+title: SafeGraph Mobility Patterns
+parent: Data Sources and Signals
+grand_parent: COVIDcast API
+---
+
+# SafeGraph Mobility Patterns
+
+* **Source name:** `safegraph_patterns`
+* **Number of data revisions since 23 June 2020:** 0
+* **Date of last change:** never
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+
+This data source uses data reported by [SafeGraph](https://www.safegraph.com/) 
+using anonymized location data from mobile phones. SafeGraph provides
+[Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) dataset
+to eligible researchers who obtain an API key. SafeGraph provides this data for
+individual census block groups, using differential privacy to protect the
```

Comment:
Grammatically we could condense:
"using differential privacy to protect the privacy of individual people in the data" to "using differential privacy to protect individual people's privacy in data" 

Not a major issue and could be left as is, but figured suggesting condensed language is always beneficial if it can be used.